components/profile_popover.jsx

```diff
@@ -320,18 +370,6 @@ export default class ProfilePopover extends React.Component {
     }
 }
 
-ProfilePopover.defaultProps = {
-    isRHS: false,
-    hasMention: false
-};
-
-ProfilePopover.propTypes = Object.assign({
-    src: PropTypes.string.isRequired,
-    user: PropTypes.object.isRequired,
-    status: PropTypes.string,
-    isBusy: PropTypes.bool,
-    hide: PropTypes.func,
-    isRHS: PropTypes.bool,
-    hasMention: PropTypes.bool
-}, Popover.propTypes);
 delete ProfilePopover.propTypes.id;
+
+export default ProfilePopover;
```

Comment:
The AST that gets generated won't recognize leading comments on the class if it includes `export default class ...` on the same line. I spent a bit of time tracking down where the bug is caused but it's in a dependency of dependency of dependency so it was easier to make this change